contrib/intel/jenkins/Jenkinsfile

```diff
@@ -122,6 +122,21 @@ pipeline {
                      }
 
                  }
+		 stage('ze-shm') {
+		     agent {node {label 'eth'}}
+		     steps{
+                        withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin/:$PYTHONPATH'])
+                        {
+                          sh """
+                            env
+                            (
+                                cd  ${env.WORKSPACE}/contrib/intel/jenkins/
+                                python3.7 runtests.py --prov=shm --ofi_build_mode='dbg'
```

Comment:
since we are testing the shm provider here we can remove it from the stage('eth-udp-shm-dbg')